js/src/Map.js

```diff
@@ -34,7 +34,6 @@ export class LeafletMapModel extends widgets.DOMWidgetModel {
       _model_module: 'jupyter-leaflet',
       _view_module: 'jupyter-leaflet',
 
-      window_url: '',
```

Comment:
Shouldn't we keep this?

Reply:
It doesn't seem to be needed.

Reply:
It is needed when having an embedded widget, as it retrieves the model from here. Anyway as we discussed this PR won't make sense in the case of an embedded widget.